android/src/main/java/io/sentry/capacitor/SentryCapacitor.java

```diff
@@ -180,17 +182,21 @@ public void setUser(PluginCall call) {
                 }
 
                 if (otherUserKeys != null) {
-                    Map<String, String> otherUserKeysMap = new HashMap<>();
+                    HashMap<String, String> otherUserKeysMap = new HashMap<>();
                     Iterator<String> it = otherUserKeys.keys();
 
                     while (it.hasNext()) {
                       String key = it.next();
                       String value = otherUserKeys.getString(key);
 
-                      otherUserKeysMap.put(key, value);
+                      // other is ConcurrentHashMap and can't have null values
+                      if (value != null) {
+                        otherUserKeys.put(key, value);
+                      }
                     }
 
-                    userInstance.setOthers(otherUserKeysMap);
+                    // Works on the Android SDK but doesn't seems to be supported by the back-end.
```

Comment:
Could you link and example event, it the data are in the processed event JSON, it's  good. Only UI issue.

Reply:
https://[us.sentry.io/api/0/projects/sentry-sdks/sentry-capacitor/events/c15bc11c29a842488d8891b37416a211/json/](https://us.sentry.io/api/0/projects/sentry-sdks/sentry-capacitor/events/c15bc11c29a842488d8891b37416a211/json/)

You can see on the json payload the user has `data":{"teeth":15}},`
where the event itself doesn't show this additional data on the user
https://sentry-sdks.sentry.io/issues/5061824231/events/351f306ce50c401694c2bb040b80546f/?project=5522756